### ISSUE TITLE
Building table rows with child components breaks table striping

### ISSUE THREAD

**Issue by cleonello** [adamghill/django-unicorn#130]
Using a parent table component and creating table rows out of child components results in alternating table_row and script tags like so:

```
<tr unicorn:id="3gyuTizE:payment-row:56" unicorn:name="payment-row" unicorn:key="56" unicorn:checksum="GnYyQM78">
...
</tr>
<script type="module">if (typeof Unicorn === 'undefined') { console.error('Unicorn is missing. Do you need {% load unicorn %} or {% unicorn-scripts %}?') } else { Unicorn.componentInit({"id":"3gyuTizE:payment-row:56",...}</script>
```

This breaks table striping as implemented in many css frameworks (tested with Bulma, Materialize, Picnic and Siimple).  Using plain css as below also does not work properly b/c of the additional script tags after every row:

```
tr:nth-child(even) {
    background-color: #F3F4F6;
}
tr:nth-child(odd) {
    background-color: #aeb2bb;
} 
```

**Comment by adamghill** [comment]
I think ideally child components wouldn't output a new script tag like it does now. They could be added to the parent's component initialization which would be cleaner anyway.

If I could get that working would that solve this issue?

**Comment by adamghill** [comment]
I created PR #131 which I think should solve this problem. I need to write some tests around this functionality, but I should be able to get it into the next release in the next few days.

**Comment by cleonello** [comment]
Awesome!  Yes, it would solve the issue.  I also agree that it would be cleaner not to output a script tag for each child component.

BTW, great work on django-unicorn!  I'm excited to try it out on a couple new projects.  Thanks for the hard work!

**Comment by adamghill** [comment]
I merged the fix into master in commit 4b5e68157d0de9e3f2a31b80aec607259319d16f.

I'm going to close this for now, but feel free to open a new issue if you run into anything else!

**Comment by adamghill** [comment]
Fix released in 0.17.2. https://www.django-unicorn.com/docs/changelog/#v0-17-2

**Comment by cleonello** [comment]
Thanks!  Just updated and seems to work as expected!
